pages/ambassadors.tsx

```diff
@@ -0,0 +1,467 @@
+                  >
+                    Receive exclusive thirdweb merch that identifies you as part
+                    of the team.
+                  </ProductCard>
+                  <ProductCard
+                    title="Network"
+                    icon={require("/public/assets/ambassadors/network-icon.svg")}
+                  >
+                    Meet like-minded individuals and grow your network in web3.
+                  </ProductCard>
+                  <ProductCard
+                    title="Access"
+                    icon={require("/public/assets/ambassadors/access-icon.svg")}
+                  >
+                    Insider access to the team, roadmap, events.
+                  </ProductCard>
+                </SimpleGrid>
+              </Flex>
+            </Flex>
+          </HomepageSection>
+          <HomepageSection>
+            <Flex
+              pt={24}
+              mb={{ base: 24, md: -24 }}
+              flexDir="column"
+              gap={{ base: 6, md: 8 }}
+              align={{ base: "initial", md: "center" }}
+            >
+              <Heading
+                as="h2"
+                fontSize={{ base: "64px", md: "64px" }}
+                fontWeight={700}
+                letterSpacing="-0.04em"
+                mb={4}
+                textAlign="center"
+              >
+                Meet the{" "}
+                <Text
+                  fontSize={{ base: "64px", md: "64px" }}
+                  fontWeight={700}
+                  letterSpacing="-0.04em"
+                  as="span"
+                  bgGradient="linear-gradient(243.9deg, #BFA3DA 21.81%, #84309C 48.81%, #C735B0 86.61%);"
+                  bgClip={"text"}
+                >
+                  Ambassadors.
+                </Text>
+              </Heading>
+              <SimpleGrid
+                columns={{ base: 2, md: 5 }}
+                gap={8}
+                justifyContent="space-evenly"
+              >
+                {ambassadors.map((ambassador) => (
+                  <Flex key={ambassador.name} flexDir="column" gap={1}>
+                    <Heading size="title.sm">{ambassador.name}</Heading>
+                    {ambassador.twitter ? (
+                      <TrackedLink
+                        href={`https://twitter.com/${ambassador.twitter}`}
+                        isExternal
+                        category="team"
+                        label={ambassador.name}
+                      >
+                        <Text size="label.md" color="gray.500">
+                          @{ambassador.twitter}
+                        </Text>
+                      </TrackedLink>
+                    ) : (
+                      <Text
+                        size="label.md"
+                        color="gray.700"
+                        fontWeight={400}
+                        fontStyle="italic"
+                      >
+                        no twitter
```

Comment:
isn't it better to just not render anything if they don't have twitter?

Reply:
feels bad we're copy pasting instead of making it it's own component if it's the same but oh well, just leave it then